tests/acceptance.suite.dist.yml

```diff
@@ -37,5 +37,5 @@ modules:
             admin email: 'admin@mydomain.com'      # email Id of the Admin
             language: 'English (United Kingdom)'   # Language in which you want the Application to be Installed
         AcceptanceHelper:
-            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/' # Path to the Extension to be installed via Install from folder
+            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/.dist/redCORE' # Path to the Extension to be installed via Install from folder
```

Comment:
I have no clue how this passed:

![screen shot 2015-05-20 at 12 34 09](https://cloud.githubusercontent.com/assets/1375475/7723801/c360c28e-feeb-11e4-9904-13d5e361caaa.png)